core/templates/dev/head/pages/header_css_libs.html

```diff
@@ -1,11 +1,4 @@
-<link rel="stylesheet" type="text/css" media="screen"
-      href="https://fonts.googleapis.com/css?family=Capriola|Rubik|Roboto|Material+Icons|Open+Sans:400,600">
-{% if not DEV_MODE %}
-  <link rel="stylesheet" type="text/css" media="screen"
-        href="/third_party/generated/css/third_party.min.css">
-{% else %}
-  <link rel="stylesheet" type="text/css" media="screen"
-        href="/third_party/generated/css/third_party.css">
-{% endif %}
-<link rel="stylesheet" type="text/css" media="screen"
-      href="/templates/dev/head/css/oppia.css">
+<link rel="stylesheet" type="text/css" media="screen" href="https://fonts.googleapis.com/css?family=Capriola|Rubik|Roboto|Material+Icons|Open+Sans:400,600">
+<link ng-if="!DEV_MODE" rel="stylesheet" type="text/css" media="screen" ng-href="/third_party/generated/css/third_party.min.css">
```

Comment:
Can you locate me to the scope where DEV_MODE is defined?

Reply:
@DubeySandeep 
![dev](https://user-images.githubusercontent.com/43509699/51389993-ff3e0180-1b53-11e9-94bc-c36465e38224.png)
Here is this definition of DEV_MODE